functime/feature_extraction/tsfresh.py

```diff
@@ -1495,7 +1530,18 @@ def variation_coefficient(x: TIME_SERIES_T) -> FLOAT_EXPR:
     -------
     float | Expr
     """
-    return x.std(ddof=0) / x.mean()
+    if isinstance(x, pl.Series):
+        if x.mean() in (0, float(0)):
+            return 0
+        else:
+            return x.std(ddof=0) / x.mean()
+    else:
+        return (
+            pl.when(x.mean() != float(0))
+            .then(x.std(ddof=0) / x.mean())
+            .otherwise(0)
+            .keep_name()
+        )
```

Comment:
This is great, thanks for this. 

Totally agree - I think I was only looking to return 0 because I thought tsfresh would, but looking back [that's not the case](https://tsfresh.readthedocs.io/en/latest/_modules/tsfresh/feature_extraction/feature_calculators.html#variation_coefficient).